interface/wx/utils.h

```diff
@@ -883,9 +882,10 @@ bool wxGetUserId(char* buf, int sz);
 /**
     This function returns the full user name (something like "Mr. John Smith").
 
-    Under Windows or NT, this function looks for the entry UserName in the
-    wxWidgets section of the WIN.INI file. If PenWindows is running, the entry
-    Current in the section User of the PENWIN.INI file is used.
+    Under Windows, this function looks for the entry UserName in the
```

Comment:
The WIN.INI part should be removed (as the useless `::GetProfileString()` was removed) and the rest changed.

BTW, does the function really return "something like "Mr. John Smith" (see the first paragraph), i.e., including stuff such as "Mr. "?

Reply:
Fixed, and expanded help. No, "Mr. " will not be included (unless you actually put that into the system as part of your first name); I removed that. Thanks!